tests/unit/test_attribute_mapping_collection.py

```diff
@@ -1,172 +1,181 @@
-        )
-
-    @responses.activate
-    def test_by_relative_id(self):
-        url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
-        responses.add(responses.GET, url, json=self.mappings_json)
-        tester = AttributeMappingCollection(self.tamr, url)
-        by_relative = tester.by_relative_id("projects/4/attributeMappings/19629-12")
-        self.assertEqual(
-            by_relative.unified_attribute_name,
-            self.mappings_json[0]["unifiedAttributeName"],
-        )
-
-    @responses.activate
-    def test_create(self):
-        def create_callback(request, snoop):
-            snoop["payload"] = request.body
-            return 200, {}, json.dumps(self.mappings_json[0])
-
-        url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
-        responses.add(responses.GET, url, json=self.mappings_json)
-        snoop_dict = {}
-        responses.add_callback(
-            responses.POST, url, partial(create_callback, snoop=snoop_dict)
-        )
-        map_collection = AttributeMappingCollection(
-            self.tamr, "projects/4/attributeMappings"
-        )
-        test = map_collection.create(self.create_json)
-        self.assertEqual(test.input_dataset_name, self.create_json["inputDatasetName"])
-        self.assertEqual(json.loads(snoop_dict["payload"]), self.create_json)
-
-    @responses.activate
-    def test_create_from_spec(self):
-        def create_callback(request, snoop):
-            snoop["payload"] = json.loads(request.body)
-            return 200, {}, json.dumps(self.mappings_json[0])
-
-        url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
-        responses.add(responses.GET, url, json=self.mappings_json)
-        snoop_dict = {}
-        responses.add_callback(
-            responses.POST, url, partial(create_callback, snoop=snoop_dict)
-        )
-
-        map_collection = AttributeMappingCollection(
-            self.tamr, "projects/4/attributeMappings"
-        )
-        spec = (
-            AttributeMappingSpec.new()
-            .with_relative_input_attribute_id(
-                self.create_json["relativeInputAttributeId"]
-            )
-            .with_input_dataset_name(self.create_json["inputDatasetName"])
-            .with_input_attribute_name(self.create_json["inputAttributeName"])
-            .with_relative_unified_attribute_id(
-                self.create_json["relativeUnifiedAttributeId"]
-            )
-            .with_unified_dataset_name(self.create_json["unifiedDatasetName"])
-            .with_unified_attribute_name(self.create_json["unifiedAttributeName"])
-        )
-        map_collection.create(spec.to_dict())
-
-        self.assertEqual(snoop_dict["payload"], self.create_json)
-
-    create_json = {
-        "relativeInputAttributeId": "datasets/6/attributes/suburb",
+@pytest.fixture
+def client():
+    from tamr_unify_client import Client
```

Comment:
could you move these imports to be at the top so that we are consistent with the other test file?

Reply:
Thanks, I missed that one.